base/combinatorics.jl

```diff
@@ -115,7 +115,7 @@ function swaprows!(a::AbstractMatrix, i, j)
     end
 end
 
-# like permute!! applied to each row of a, in-place in a (overwriting p).
+# like permute! applied to each row of a, in-place in a (overwriting p).
 function permutecols!!(a::AbstractMatrix, p::AbstractVector{<:Integer})
```

Comment:
What about this one? Can we remove it too?

Reply:
There are 2 use cases:
https://github.com/JuliaLang/julia/blob/0b8e8fc216abc4e93a11279a1a0a5b4b0bda23fa/stdlib/LinearAlgebra/src/eigen.jl#L132-L136

https://github.com/JuliaLang/julia/blob/0b8e8fc216abc4e93a11279a1a0a5b4b0bda23fa/stdlib/LinearAlgebra/src/symmetric.jl#L677-L681

Maybe we could move `permutecols!!` into the LinearAlgebra package.